android/app/src/main/java/org/fossasia/openevent/OpenEventApp.java

```diff
@@ -106,6 +109,12 @@ public void onCreate() {
         }
         refWatcher = LeakCanary.install(this);
 
+        //Initialize Cache
+        File httpCacheDirectory = new File(getCacheDir(), "picasso-cache");
+        Cache cache = new Cache(httpCacheDirectory, 15 * 1024 * 1024);
+
```

Comment:
thanks for pointing that out. 
but i am still doubtful that every device will allow a cache file of 10MB. Lets say if its full then there is no point caching right?